Library/.rubocop.yml

```diff
@@ -8,7 +8,17 @@ AllCops:
 
 require: ./Homebrew/rubocops.rb
 
+Homebrew/FormulaCop:
+  Enabled: false
+
 Homebrew/CorrectBottleBlock:
+  Include:
+    - '**/Taps/homebrew/**/*.rb'
```

Comment:
I think just using `'**/Taps/**/*.rb'` may be sufficient? We don't want these applying to non-`HOMEBREW_LIBRARY_PATH` Ruby files.